hack/boilerplate/boilerplate.sh.txt

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# Copyright 2020 Google LLC All Rights Reserved.
+# Copyright 2020 KO Build Authors All Rights Reserved.
```

Comment:
```suggestion
# Copyright 2020 ko Build Authors All Rights Reserved.
```

It's a silly styling thing, but usually it's lowercase.

(here and in boilerplate.go.txt)

Reply:
roger that

Reply:
done PTAL